### ISSUE TITLE
Trial Balance report should reflect only 0 as opening balances for all Income and Expense GLs 

### ISSUE THREAD

**Issue by Swati8848** [frappe/erpnext#41453]
### Information about bug

As per Accounting rules, there is no opening balance for income and expense accounts. All the balances has to be moved to P&L A/c when we close the books. Trial Balance report should reflect only 0 as opening balances for all Income and Expense GLs as all the balances should get transferred to P&L A/c GL in the period closing voucher. Opening balances shouldn't change if we check or uncheck period closing entry check box.

### Module

accounts

### Version

Frappe Version 14
ERPNEXT Version 14

### Installation method

None

**Comment by rohitwaghchaure** [comment]
Did you create Period Closing Voucher for the last FY?
https://docs.erpnext.com/docs/user/manual/en/period-closing-voucher

**Comment by Swati8848** [comment]
Yes after posting period closing voucher for 2023-2024, we are checking
opening trial balance for 2024-2025.

**Comment by saivineeth100** [comment]
> Did you create Period Closing Voucher for the last FY? https://docs.erpnext.com/docs/user/manual/en/period-closing-voucher

Irrespective of closing vouchers,
Income and Expense accounts (where report_type="Profit and Loss") should not have any opening or closing balances

**Comment by DipenFrappe** [comment]
<img width="1280" height="715" alt="Image" src="https://github.com/user-attachments/assets/d2ef8918-691e-4ebc-92a5-0dcd0f888cd9" />
